app/src/main/java/co/nimblehq/compose/crypto/ui/screens/home/HomeScreen.kt

```diff
@@ -218,6 +260,7 @@ private fun MyCoins(
     }
 }
 
+@SuppressLint("UnrememberedMutableState")
```

Comment:
I don't see any issue without this using annotation. Could we get rid of it?

<img width="787" alt="image" src="https://user-images.githubusercontent.com/16315358/200467896-4483b3f1-00ea-4bf9-819d-c3c6bc0f3e93.png">